keylime-agent/src/secure_mount.rs

```diff
@@ -163,11 +146,9 @@ mod tests {
 
     #[test]
     fn test_secure_mount() {
-        let path = "/var/lib/keylime";
-        let work_dir = Path::new(&path);
-        let secure_dir_path = Path::new(work_dir).join("secure");
+        let temp_workdir = tempfile::tempdir().unwrap(); //#[allow_ci]
```

Comment:
I suggest you keep the `"secure"` to force `mount()` to create the directory. It will increase the coverage. Something like:

```suggestion
        let temp_workdir = tempfile::tempdir().unwrap(); //#[allow_ci]
        let secure_dir_path = Path::new(temp_workdir).join("secure");
```

Reply:
Actually it seems that this decreased code coverage a lot from point of secure mount.